docs/deployment.md

```diff
@@ -3,6 +3,14 @@ sidebar_position: 8
 title: "☁️ Deployment"
 ---
 
+:::caution Alpha Status
+Chroma Server is currently in Alpha.
```

Comment:
What does this mean to users tangibly?

Been saying something like this 
"We are working hard to move chroma from a in-memory single-process oriented library to a distributed production-grade db!"